docs/en_US/Tutorial/InstallCustomizedAlgos.rst

```diff
@@ -160,3 +160,9 @@ Run following command to uninstall an installed package:
 For example:
 
 ``nnictl algo unregister demotuner``
+
+
+Migrate previous customized algorithms
```

Comment:
Porting customized algorithm worked on v1.X to v2.X

Reply:
fix it